### PR TITLE
Fix #13905: Wrong colours in Playback Setup

### LINKED ISSUE #13905: Wrong colour definition used for the 'Active' text in 'Playback Setup' dialog
**Describe the bug**
In the light theme, the word 'Active' can not be read since the colour definition appears to be wrong

image

**Fix**
It should be using **FONT_PRIMARY_COLOUR**, set to 70% opacity

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #13905

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Tantacrul** [comment]
OK, one more comment. I got a little confused by the values you sent me before. I think it should actually be much simpler.

Basically, the 'Active' and 'Inactive' text should always be 70%. They shouldn't change depending on whether or not the are active / inactive.

The reason is that we want users to be able to read that it is inactive! So setting it the 30% fights that goal :D

Sorry @HemantAntony!

All you have to do is just share a screengrab of the next PR and I'll approve it then. No need for me to test again, since the colour values are fine. It's just that transparency setting.

**Comment by cbjeukendrup** [comment]
@Tantacrul Tangentially related, but maybe the cause of the confusion: I believe that some people are confused about the meaning of "inactive". It means "not currently selected for the current score", but people might think it means "not installed". 

How about dropping the word "inactive" and only showing the word "active" for the currently activated profile? This would make sense to me because only one can be active at a time. 
Also, we also have that nice big checkmark for the currently selected profile. Doesn't that make the words "active" and "inactive" completely redundant? 

We might also want to add a special indication for non-installed profiles (i.e. Muse Sounds when you have not installed it), to avoid any confusion.

**Comment by HemantAntony** [comment]
@Tantacrul the reason why I didn't make it 70% for both is because the text 'inactive' sticks out like this
image
'Inactive' is more bolder than 'Muse Sounds' here

Are you sure this is what you want?

Also, I do agree the words 'active' and 'inactive' is a bit confusing. Unless I have understood something wrong, Isn't a simple check box enough? similar to Theme selection:

image

**Comment by cbjeukendrup** [comment]
@HemantAntony I think you should use different logic: just use 0.7 for the active/inactive label, and multiply it with ui.theme.itemOpacityDisabled when it is disabled. It should not depend on `isActive`.

**Comment by Tantacrul** [comment]
@HemantAntony - OK, we might be talking at cross purposes here.

If MuseSounds is not installed, then the option is disabled and **in that case** yes, the opacity for both should be 30%

However, when MuseSounds IS installed, none of that text should ever be 30% (whether active or inactive)

**Comment by Tantacrul** [comment]
I'm aware I've used the word 'disabled' here which is probably better :)

File: src/playback/qml/MuseScore/Playback/SoundProfilesDialog.qml [hunk 1]
```diff
@@ -146,7 +146,8 @@ StyledDialogView {
                                 StyledTextLabel {
                                     Layout.preferredWidth: titleColumn.width
 
-                                    color: ui.theme.fontSecondaryColor
+                                    color: ui.theme.fontPrimaryColor
+                                    opacity: isActive ? 0.7 : 0.3
```
[the remark this answers is not in the document]
Comment (vpereverzev) [reply src/playback/qml/MuseScore/Playback/SoundProfilesDialog.qml:150]: @HemantAntony any progress on this comment?

**Comment by Tantacrul** [comment]
Hey Hemant. Is it possible you can find time to finalise this one. Basically, the only issue is that the word **Inactive** is too transparent. In this example, it should be using the same opacity as **Active**

image

**Comment by HemantAntony** [comment]
@vpereverzev @Tantacrul So sorry for the delay :)

**Comment by Tantacrul** [comment]
OK, I think this is good to go now. Thanks @HemantAntony.

@RomanPudashkin @Eism - if you're happy, can this patient be prescribed with 11 CCs of **merge**?

**Pull request merged**